other/require-replicas-allow-disruption/require-replicas-allow-disruption.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: require-replicas-allow-disruption
+  annotations:
+    policies.kyverno.io/title: Require Replicas Allow Disruption
+    policies.kyverno.io/category: Other
+    kyverno.io/kyverno-version: 1.11.4
+    kyverno.io/kubernetes-version: "1.27"
+    policies.kyverno.io/subject: PodDisruptionBudget, Deployment
+    policies.kyverno.io/description: >-
+      Existing PodDisruptionBudgets can apply to all future matching Pod controllers.
+      If the minAvailable field is defined for such matching PDBs and is lower than the
+      replica count of a new Deployment, then availability could be negatively impacted.
+      This policy specifies that Deployment replicas exceed the minAvailable value of all
+      matching PodDisruptionBudgets which specify minAvailable as a number and not percentage.
+spec:
+  validationFailureAction: Audit
+  background: false
+  rules:
+  - name: replicas-check
+    match:
+      any:
+      - resources:
+          kinds:
+          - Deployment
```

Comment:
Should probably add StatefulSet as well. The structure in this area is the same, so it comes for "free".

Reply:
Typically I like making the match as simple as possible to "illustrate" the use case and rely on the user to add others based on their specific needs. I don't mind adding just one other in this instance though.

Reply:
Makes sense. Somehow it forces the user to understand the policy - which is good! 👍

Reply:
Done.